### ISSUE TITLE
Redesign Flow of Starting a Game from Menu

### ISSUE THREAD

**Issue by timblegoorn** [cs481-ekh/f22-kitty-cat-studios#368]
While though we have done a good job implementing various new features such as difficulty selection, freeplay, and runner selection, we do not have a way to combine these selections. Thus, I will be redesigning the way games are started from the menu. In essence, the flow will go as such:

Main menu has a "Play" button. Clicking play button leads to a "game mode select" screen, here the user can choose to play standard, freeplay (practice), or survival (to be implemented). After selecting a gamemode, the user is prompted for which map they would like to play on (for survival and freeplay), followed by runner selection, and lastly difficulty selection. At any point in this process the user can go back to the previous screen or back to the main menu. Once user selects a difficulty, the game will start with all selected settings and configurations.

TLDR; Play -> Gamemode -> Choose map (if applicable) -> Choose Runner -> Choose Difficulty -> Game starts

- [ ] #376
- [ ] #377
- [ ] #378
- [ ] #379

**Comment by timblegoorn** [comment]
Finally finished it up! All in all I just plugged away at this story and was able to get it nailed together. Really proud of the changes and think it makes a big difference. Approximately 7-8ish hours spent on this story
